pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
Sorry to be naive on this, but I use `--fasttest` all the time, and my assets are always there. Do I have that wrong? Reading the code, I must misunderstand, but I'm not sure how I could've run tests...